src/elements/tabs/tab-label/tab-label.component.ts

```diff
@@ -16,6 +18,7 @@ import style from './tab-label.scss?lit&inline';
  * @slot - Use the unnamed slot to add content to the tab title.
  * @slot icon - Use this slot to display an icon to the left of the title, by providing the `sbb-icon` component.
  * @slot amount - Provide a number to show an amount to the right of the title.
+ * @event {Event} active - The `active` event fires when the tab has been activated via user selection.
```

Comment:
```suggestion
 * @event {Event} active - The `active` event fires when the connected tab has been activated via user selection.
```
for user convenience maybe having it on the tab as well would be better? Still not 100% what is best :D